README.md

```diff
@@ -123,3 +123,4 @@ in the online documentation for an overview on how to configure Spark.
 
 Please review the [Contribution to Spark guide](https://spark.apache.org/contributing.html)
 for information on how to get started contributing to the project.
+##
```

Comment:
hmm... why we need this change?

Reply:
I had trouble retriggering the job after enabling github actions on my fork, I'll remove this line again